src/networking.c

```diff
@@ -209,6 +210,91 @@ cc_oci_tap_create(const gchar *const tap) {
 }
 
 /*!
+ * Helper function for setting/getting MTU for network interface.
+ *
+ * \param ifname Interface name.
+ * \param mtu MTU to be set/retrieved for the interface.
+ * \param set Flag indicating if the mtu needs to be set/retrieved.
+ *
+ * \return \c true on success, else \c false.
+ */
+gboolean
+cc_oci_handle_interface_mtu(char *ifname, unsigned int *mtu, gboolean set)
```

Comment:
@amshinde It could be github playing tricks with me, but I don't see a check on `mtu` being non-NULL in the diffs I'm looking at or in the branch in your repo - can you check from your side that you have pushed the changes etc.  thx